packages/contract/script/pay/DeployDaimoPayBridger.s.sol

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.13;
+
+import "forge-std/Script.sol";
+import "openzeppelin-contracts/contracts/proxy/ERC1967/ERC1967Proxy.sol";
+
+import "../../src/pay/DaimoPayBridger.sol";
+import "../Constants.s.sol";
+
+contract DeployDaimoPayBridger is Script {
+    function run() public {
+        vm.startBroadcast();
+
+        // initOwner = daimo.eth
+        address initOwner = 0xEEee8B1371f1664b7C2A8c111D6062b6576fA6f0;
+
+        DaimoPayBridger implementation = new DaimoPayBridger{salt: 0}(
+            initOwner
+        );
+
+        (
+            uint256[] memory chainIds,
+            address[] memory bridgers
+        ) = _getBridgersAndChainIds();
+
+        address bridger = CREATE3.deploy(
+            keccak256("DaimoPayBridger-test1"),
+            abi.encodePacked(
+                type(ERC1967Proxy).creationCode,
+                abi.encode(
+                    address(implementation),
+                    abi.encodeWithSelector(
+                        DaimoPayBridger.init.selector,
+                        chainIds,
+                        bridgers
+                    )
+                )
+            )
+        );
+
+        vm.stopBroadcast();
+
+        console.log("bridger deployed at address:", bridger);
+    }
+
+    function _getBridgersAndChainIds()
+        private
+        view
+        returns (uint256[] memory chainIds, address[] memory bridgers)
+    {
+        bool testnet = _isTestnet(block.chainid);
+
+        address cctpBridger = CREATE3.getDeployed(
+            msg.sender,
+            keccak256("DaimoPayCCTPBridger-test1")
+        );
+        address acrossBridger = CREATE3.getDeployed(
+            msg.sender,
+            keccak256("DaimoPayAcrossBridger-test1")
+        );
+
+        if (testnet) {
+            // TODO
+            return (chainIds, bridgers);
```

Comment:
👍 
could do just cctp on testnet, Sepolia <> Base Sepolia, but even that is optional